packages/x-components/package.json

```diff
@@ -104,7 +104,7 @@
     "@vue/test-utils": "~1.0.3",
     "autoprefixer": "~9.7.4",
     "commitizen": "~4.0.3",
-    "cypress": "~7.7.0",
+    "cypress": "~8.4.1",
```

Comment:
Revert the changes of this file please, we only need the ones in the lock files.!